src/engine.rs

```diff
@@ -244,6 +256,15 @@ mod tests {
 
     #[test]
     fn test_js_function() -> Result<()> {
+        let engine: Engine = Engine::new(
+            &Config::new()
+                .wasm_multi_memory(true)
+                .wasm_threads(false)
+                .consume_fuel(true)
+                .epoch_interruption(true),
+        )?;
```

Comment:
All of these _should_ be the same, but it's possible that they get out of sync in the future (e.g. someone updates them and forgets to update one). Should we have a helper function to keep it DRY?